detect_secrets/plugins/keyword.py

```diff
@@ -80,53 +81,81 @@
     "\\k.*'",
     '`cat',
     '`grep',
+    '`sudo',
     'account_password',
+    'api_key',
+    'disable',
     'dummy_secret',
     'dummy_value',
     'false',
     'false):',
     'false,',
     'false;',
+    'login_password',
     'none',
     'none,',
     'none}',
     'not',
+    'not_real_key',
     'null',
     'null,',
     'null.*"',
     "null.*'",
     'null;',
+    'pass',
+    'pass)',
     'password',
     'password)',
+    'password))',
```

Comment:
Just to clarify, does this mean a word that contains `password))` will be marked as a false-positive?

Reply:
The short-answer is, only if it `==` `password))`, not contains.

The `FALSE_POSITIVES` set is for RHS values normally resulting in false-positives, you could make your real password `password))`, but it's more likely that e.g. I made some mistake in not requiring quotes for that particular filetype, or haven't made a custom regex for that language when I should, this set patches over certain things like that.